slpp.py

```diff
@@ -101,6 +98,18 @@ def white(self):
             else:
                 break
 
+        self.skip_comments()
+
+    def skip_comments(self):
+        if self.ch == '-' and self.text[self.at] == '-':
+            # `--` is a comment, skip to next new line
+            while self.ch:
+                if re.match('\n', self.ch):
+                    self.white()
+                    break
+                else:
+                    self.next_chr()
+
```

Comment:
There is also the `--[[` and `]]--` comment syntax, which doesn't look covered

Reply:
I added support for `--[[ ]]` in https://github.com/SirAnthony/slpp/commit/232cbf0960a782719d81a311ce8cbaaafd6b77a4
All other fancy comments like `--[[  --[=[]] --]=]` or any combinations of = is not covered also, but if you put multilevel comments with executionable code in texts/configs, you doing something wrong imho. Pull requests are welcome, though.